cli/plugins/builder/README.md

```diff
@@ -0,0 +1,20 @@
+[Flogo CLI](https://github.com/project-flogo/cli) Plugin for Flogo Enterprise Builder
+
+
+Preparation:
+
+     1. Install Flogo CLI first, please following CLI guideline
+     2. Set FLOGO_HOME Env Variable, such as: 
+        export FLOGO_HOME=/opt/tibco/flogo/2.6
+
+Installation
+    
+      1. Install plugin: flogo plugin install github.com/TIBCOSoftware/tci-flogo/cli/plugins/builder
```

Comment:
I think ```github.com/TIBCOSoftware/tci-flogo/project-flogo/cli/plugin``` would be better. @rishikeshpalve ??

Reply:
Changed to  github.com/TIBCOSoftware/tci-flogo/project-flogo/cli/plugins/builder